### PR TITLE
Update PLA Pokemon data to correct values

### PR DESCRIPTION
**Pull request by kittenchilly**
## Description
Just updates our PLA Pokemon data by cross referencing it with a datamine. It should be almost completely accurate now.

## **Discord contact info**
kittenchilly

### CODE REVIEW

**Comment by AlexOn1ine** [comment]
Where is the data coming from? I checked Basculin Red Striped and it has a 25 Catch rate on [Pokemondb](https://pokemondb.net/pokedex/basculin)

**Comment by kittenchilly** [comment]
> Where is the data coming from? I checked Basculin Red Striped and it has a 25 Catch rate on [Pokemondb](https://pokemondb.net/pokedex/basculin)

https://gist.github.com/kwsch/d8aea4f1c762ee0d4db8599917fe6b48
This has data for catch rate, and it says its 190 for all Basculin forms.

File: src/data/pokemon/species_info.h [hunk 1]
```diff
@@ -21017,11 +21018,11 @@ const struct SpeciesInfo gSpeciesInfo[] =
         .baseSpAttack  = 45,
         .baseSpDefense = 70,
         .types = { TYPE_BUG, TYPE_ROCK},
-        .catchRate = 25,
+        .catchRate = 15,
         .expYield = 175,
         .evYield_Attack    = 2,
```
Comment (AlexOn1ine) [inline src/data/pokemon/species_info.h:21023]: Kleavor yiealds 3 attack evs

File: src/data/pokemon/species_info.h [hunk 2]
```diff
@@ -21061,15 +21062,15 @@ const struct SpeciesInfo gSpeciesInfo[] =
         .baseSpAttack  = 80,
         .baseSpDefense = 75,
         .types = { TYPE_WATER, TYPE_GHOST},
-        .catchRate = 25,
-        .expYield = 186,
+        .catchRate = 45,
+        .expYield = 265,
         .evYield_HP     = 2,
```
Comment (AlexOn1ine) [inline src/data/pokemon/species_info.h:21067]: Yields 3 HP evs.

File: src/data/pokemon/species_info.h [hunk 3]
```diff
@@ -21083,15 +21084,15 @@ const struct SpeciesInfo gSpeciesInfo[] =
         .baseSpAttack  = 100,
         .baseSpDefense = 75,
         .types = { TYPE_WATER, TYPE_GHOST},
-        .catchRate = 25,
-        .expYield = 186,
+        .catchRate = 45,
+        .expYield = 265,
         .evYield_HP     = 2,
```
Comment (AlexOn1ine) [inline src/data/pokemon/species_info.h:21089]: Yields 3 HP evs.

File: src/data/pokemon/species_info.h [hunk 4]
```diff
@@ -21105,13 +21106,13 @@ const struct SpeciesInfo gSpeciesInfo[] =
         .baseSpAttack  = 40,
         .baseSpDefense = 80,
         .types = { TYPE_FIGHTING, TYPE_POISON},
-        .catchRate = 135,
-        .expYield = 179,
+        .catchRate = 20,
+        .expYield = 102,
         .evYield_Attack    = 1,
         .evYield_Speed     = 1,
```
Comment (AlexOn1ine) [inline src/data/pokemon/species_info.h:21112]: Only 2 attack evs

File: src/data/pokemon/species_info.h [hunk 5]
```diff
@@ -23894,7 +23896,7 @@ const struct SpeciesInfo gSpeciesInfo[] =
         .baseSpAttack  = 80,
         .baseSpDefense = 55,
         .types = { TYPE_WATER, TYPE_WATER},
-        .catchRate = 25,
+        .catchRate = 190,
```
Comment (AlexOn1ine) [inline src/data/pokemon/species_info.h:23899]: The catch rate seems to be 75

**Review by AlexOn1ine — CHANGES_REQUESTED** [review]
It looks like `STANDARD_FRIENDSHIP` is currently set to 70 but all of the mons I looked at had it at 50

File: src/data/pokemon/species_info.h (hunk 5 above)
Comment (kittenchilly) [reply src/data/pokemon/species_info.h:23899]: The data I posted earlier has it at a catch rate of 190.

**Comment by kittenchilly** [comment]
> It looks like `STANDARD_FRIENDSHIP` is currently set to 70 but all of the mons I looked at had it at 50.
> 
> /Edit: Sorry, all of my reviews are messed up because I didn't know how to properly do them.

Yes, I was thinking we'd update STANDARD_FRIENDSHIP in another PR. Gen 9 mons pr has the same problem as well so I think I'd like to get it all in one swoop

File: src/data/pokemon/species_info.h (hunk 5 above)
Comment (kittenchilly) [reply src/data/pokemon/species_info.h:23898]: Also, Bulbapedia has the catch rate as 190 as well.

**Comment by kittenchilly** [comment]
All reviews applied.

**Review by AlexOn1ine — APPROVED** [review]

**Comment by AlexOn1ine** [comment]
This would probably make more sense in master, right?

**Pull request merged**